android/run_tests.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+set -eux
+
+PYTORCH_DIR="$(cd $(dirname $0)/..; pwd -P)"
+PYTORCH_ANDROID_DIR=$PYTORCH_DIR/android
+
+echo "ANDROID_HOME:$ANDROID_HOME"
+if [ ! -z "$ANDROID_HOME" ]; then
+  echo "ANDROID_HOME not set; please set it to Android sdk directory"
+fi
+
+if [ ! -d $ANDROID_HOME ]; then
+  echo "ANDROID_HOME not a directory; did you install it under $ANDROID_HOME?"
+  exit 1
+fi
+
+echo "ANDROID_NDK:$ANDROID_NDK"
```

Comment:
similar here: where is ANDROID_NDK used? do you do it for gradle?

Reply:
Yes, the same.
Gradle needs it to build native libs. It can be specified as ANDROID_NDK or ndk.dir in gradle.properties